src/pipelines.js

```diff
@@ -276,7 +373,7 @@ class TextGenerationPipeline extends Pipeline {
             let startText = texts[i].trim();
             let decoded = this.tokenizer.batch_decode(outTokens, {
                 skip_special_tokens: true,
-            }).map(x => {
+            }).map((x) => {
```

Comment:
(simply back to original to not change things besides JSDoc)

```suggestion
            }).map(x => {
```